CHANGELOG.md

```diff
@@ -6,6 +6,13 @@
 ## New syntaxes
 ## New themes
 ## `bat` as a library
+
+- Add APIs to provide `Input` descriptions with `InputDescription` (@eth-p)
+- Add function to directly provide `Input`s to `PrettyPrinter` (@eth-p)
+- <font color="red">`Input::theme_preview_file` is no longer available.</font> (@eth-p)
+
+Changes colored <font color="red">red</font> are breaking changes.
```

Comment:
Unfortunately, GitHub Markdown removes these `<font>` tags. Maybe we just use something like:
* **Breaking**: …